### PR TITLE
Add envelopes

### LINKED ISSUE #22535: Feature request: Functional envelopes
## Description

We have envelope sprites in the game but they aren't functional. I believe will help deliver important notes meant for specific individuals (perhaps being only able to open the package with your id card/PDA similar to how delta-v does it) and you could write who it's from and even stamp it for legitimacy. It would also have a lot of opportunity for roleplay too, like having to deliver top-secret documents.

**Screenshots**

image

### PR DESCRIPTION
**Pull request by themias**
## About the PR

Envelopes can hold one paper document. They can be written on like paper to add a recipient/sender. 
Stamps can be used to prevent the writing from being edited. (We hide the stamp layer on the item since it doesn't line up.)

Sealing and tearing have a small doafter. Envelopes cannot be reused once torn.

Closes #22535

Future TODO: get the clown/approved/denied stamps to line up on the envelope, also fix whatever causes multiple stamps to mess up the margins.

## Why / Balance

As mentioned in the issue, this would add some optional bureaucracy gameplay, as well as roleplaying opportunities for delivering secret notes, etc.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

https://github.com/user-attachments/assets/f5e2f06b-7d30-4ba1-b398-719ca098217e

**Changelog**
:cl:
- add: Added envelopes to the PTech and bureaucracy crate

### CODE REVIEW

File: Resources/Locale/en-US/paper/envelope.ftl [hunk 1]
```diff
@@ -0,0 +1,13 @@
+envelope-description = A small envelope to keep the station's prying eyes off of your sensitive documents.
```
Comment (lzk228) [inline Resources/Locale/en-US/paper/envelope.ftl:1]: this string isnt even used anywhere

**Comment by JIPDawg** [comment]
one step closer to mailman role.

**Comment by Plykiya** [comment]
envelope-default-message....

**Comment by themias** [comment]
> envelope-default-message....

Yea, seems to be an existing bug. It happens with books or any paper that preloads a loc string. We can remove it if need be.

**Comment by Plykiya** [comment]
> > envelope-default-message....
> 
> Yea, seems to be an existing bug. It happens with books or any paper that preloads a loc string. We can remove it if need be.

It's fine, I'll fix the bug.

**Comment by ElPidor** [comment]
IF THERE'S A MAIL
THERE SHOULD BE A PIPEBOMB IN THE MAIL
God i hope you can put some c4 in it :trollface:

**Comment by he1acdvv** [comment]
It would be cool if you could add poisonous ricin and then give it to captain as "fan mail"

**Pull request merged**